### ISSUE TITLE
XOrgConfCreator fails when asked for advanced settings

### ISSUE THREAD

**Issue by ablot** [Psychtoolbox-3/Psychtoolbox-3#348]
`XOrgConfCreator` fails when asked for advanced settings (see below). Adding a `~isempty` before the `strfind` solves the problem.

Configuration:
- matlab 2016a
- 3.0.12 - Flavor: Debian package - psychtoolbox-3 (3.0.12.20160514.dfsg1-1~nd16.04+1)

Traceback:
```
Do you want to configure special/advanced settings? [y/n] y
Sorry, something went wrong. Aborting. The error message was:
Operands to the || and && operators must be convertible to logical scalar values.

Error in XOrgConfCreator (line 298)
    if ~strcmp(xdriver, 'modesetting') && strfind(winfo.GLVersion, 'Mesa') && (bitand(winfo.SpecialFlags, 2^24) > 0)
```

**Comment by kleinerm** [comment]
Yep. Fixed in my master branch since a while, but not yet released.

**Comment by ablot** [comment]
Sorry. I just checked this repo. I'll have a look at yours next time I have a problem.

**Comment by kleinerm** [comment]
No reason to be sorry - just a fyi that a fix is already ready :).

**Comment by kleinerm** [comment]
Should be fixed in the just released PTB 3.0.13.
